routers/web/base.go

```diff
@@ -126,8 +127,9 @@ func Recovery() func(next http.Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
 			defer func() {
 				if err := recover(); err != nil {
-					combinedErr := fmt.Sprintf("PANIC: %v\n%s", err, string(log.Stack(2)))
-					log.Error("%v", combinedErr)
+					common.UpdateContextHandlerPanicError(req.Context(), err)
+					combinedErr := fmt.Sprintf("PANIC: %v\n%s", err, log.Stack(2))
+					log.Error("panic occurred and gets recovered, %v", combinedErr)
```

Comment:
What does recovered here mean?

It's a panic just log it as a PANIC. This "panic occurred and gets recovered" adds nothing but make it harder to see the PANIC.

```suggestion
					log.Error("%v", combinedErr)
```

Reply:
Sometimes the code may just print the error and panic again to pass the error to caller. So I think this message makes it clear that the panic recovery stops here.